docker-wrappers/SPRAS/spras_profile/config.yaml

```diff
@@ -0,0 +1,11 @@
+jobs: 30
+executor: htcondor
+configfile: example_config.yaml
+shared-fs-usage: none
+default-resources:
+  job_wrapper: "spras.sh"
+  # If running in CHTC, this only works with apptainer images
+  container_image: "spras.sif"
+  universe: "container"
+  request_disk: "16GB"
+  request_memory: "8GB"
```

Comment:
Per our conversations, we don't know of any way to increase resources for a single job at this point, right? We could modify the Snakefile to modify resources for a single rule, but that would override resources of all instances of that rule. In SPRAS, we may only want the override to apply for certain wildcard values when that rule is run (e.g. one pathway reconstruction algorithm).

Reply:
Unfortunately, that's correct. I haven't figured out a way to apply a set of resource requirements to a job without scoping it to the rule level.

Reply:
We discussed how an advanced user could turn off the resource intensive algorithms and post-processing during an initial run, then enable the remaining algorithms after increasing resources, then enable post-processing. That is complex and we won't document it at this point.